src/biogeochem/CNCStateUpdate3Mod.F90

```diff
@@ -8,11 +8,13 @@ module CNCStateUpdate3Mod
   use shr_kind_mod                   , only : r8 => shr_kind_r8
   use shr_log_mod                    , only : errMsg => shr_log_errMsg
   use abortutils                     , only : endrun
-  use clm_time_manager               , only : get_step_size
+  use clm_time_manager               , only : get_step_size,is_end_curr_year
```

Comment:
`is_end_curr_year` is not used in `CNCStateUpdate3Mod.F90` so we don't need it here. Please remove it from the use statement.

Reply:
done